Extension/CloudFoundryCLI/cloudfoundrycli.ts

```diff
@@ -1,18 +1,15 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-/// <reference path="../../definitions/node.d.ts"/>
-/// <reference path="../../definitions/Q.d.ts" />
-/// <reference path="../../definitions/vsts-task-lib.d.ts" />
-
 import tl = require('vsts-task-lib/task');
 import path = require('path');
 import fs = require('fs');
+import process = require('process');
 import Q = require('q');
 
 var onError = function(errMsg) {
     tl.error(errMsg);
-    tl.exit(1);
+    process.exit(1);
```

Comment:
You should be able to remove the tl.exit()/process.exit() since in the code, error only seems to happens at the end. No need to kill the process, we can let is shutdown gracefully.